### PR TITLE
Troubleshooting guide: Add help link to change encoding scheme #7274

### LINKED ISSUE #7274: Troubleshooting guide: Add help link to change encoding scheme
Quite a few people have been facing issues trying to get the encoding scheme right, despite having all the project specific properties as per our `ide-usage.md`. 

[This link](https://z0ltan.wordpress.com/2011/12/25/changing-the-encoding-in-eclipse-to-utf-8-howto/) seems to have solved their issues successfully. Add this to the `troubleshooting-guide.md` under `Common setup errors and solutions`

### PR DESCRIPTION
**Pull request by RaghavBatra**
Fixes #7274 

**Outline of Solution**

Wrote what was required in the documentation along with the required solution link

### CODE REVIEW

File: docs/troubleshooting-guide.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Note that some of the screenshots might be outdated, but the instructions will r
 * **ERROR (on Windows)**: Dev server launched by Eclipse keeps running even after closing Eclipse. After restarting Eclipse, you will be able to relaunch dev server on the same port but requests will be received by the previous server instance.
 
   **REASON**: If Eclipse crashes while dev server is running inside Eclipse, the server might keep running even after Eclipse is closed.
-
+
```
Comment (VamsiSangam) [inline docs/troubleshooting-guide.md]: This seems like an unrelated change. Can you revert this?

File: docs/troubleshooting-guide.md [hunk 2]
```diff
@@ -46,7 +46,12 @@ Note that some of the screenshots might be outdated, but the instructions will r
 * **ERROR**: Eclipse complains "NewClass cannot be resolved to a type", "The import some.package.NewClass cannot be resolved", or "The method someMethod() from the type ExistingClass refers to the missing type NewClass" after syncing with `master` branch.
 
   **SOLUTION**: This is likely because the dependencies have changed. Refer to [this document](dependencies.md) for steps to update your local dependencies configuration.
-
+  
+ * **ERROR**: Issues trying to get the encoding scheme right, despite having all the project specific properties as per `ide-usage.md`.
```
Comment (VamsiSangam) [inline docs/troubleshooting-guide.md]: Change this to -

> **ERROR** - Some characters are displayed incorrectly in browser. Example, `Charlés's` is displayed as `CharlÃ©s`.

File: docs/troubleshooting-guide.md [hunk 3]
```diff
@@ -46,7 +46,12 @@ Note that some of the screenshots might be outdated, but the instructions will r
...
+
```
Comment (VamsiSangam) [inline docs/troubleshooting-guide.md]: Add a reason clause here like this -

> **REASON** - Page encoding is not set to `UTF-8`.

File: docs/troubleshooting-guide.md [hunk 4]
```diff
@@ -46,7 +46,12 @@ Note that some of the screenshots might be outdated, but the instructions will r
 * **ERROR**: Eclipse complains "NewClass cannot be resolved to a type", "The import some.package.NewClass cannot be resolved", or "The method someMethod() from the type ExistingClass refers to the missing type NewClass" after syncing with `master` branch.
 
   **SOLUTION**: This is likely because the dependencies have changed. Refer to [this document](dependencies.md) for steps to update your local dependencies configuration.
-
+  
+ * **ERROR**: Issues trying to get the encoding scheme right, despite having all the project specific properties as per `ide-usage.md`.
+ 
+   **SOLUTION**: [Click here](https://z0ltan.wordpress.com/2011/12/25/changing-the-encoding-in-eclipse-to-utf-8-howto/
```
Comment (VamsiSangam) [inline docs/troubleshooting-guide.md]: Solution clause can be written as -

> **SOLUTION** - Go to `Eclipse` → `Preferences` →  `Resources` → change the `Text file encoding` setting from `Default` to `Other: UTF-8`. If you are a Windows user, then you can navigaye to `Preferences` by `Window` → `Pereferences`. If this change does not solve the error, then you can try the methods in [this](https://z0ltan.wordpress.com/2011/12/25/changing-the-encoding-in-eclipse-to-utf-8-howto/) link.

File: docs/troubleshooting-guide.md [hunk 5]
```diff
@@ -46,7 +46,12 @@ Note that some of the screenshots might be outdated, but the instructions will r
...
+) for a solution that has solved this issue for a lot of people.
```
Comment (VamsiSangam) [inline docs/troubleshooting-guide.md]: The solution should be in a single line.

**Comment by VamsiSangam** [comment]
Although this PR seems fine, your fork's master branch is 2 commits ahead of `TEAMMTES:master`. Ideally, you should not make any changes in your fork's master branch. It should always be even/behind `TEAMMATES:master`. Please revert the commits made in your fork's master branch.
 
You need to use `git reset --hard <COMMIT_SHA>`. Try to dig a little about `git reset`. Its easy. Let me know if you have any trouble.

**Comment by RaghavBatra** [comment]
I created a new pull request: https://github.com/TEAMMATES/teammates/pull/7925

Please close this one!
